web/src/pages/index.js

```diff
@@ -371,6 +394,14 @@ function Home() {
                     >
                       Node.js & TypeScript support
                     </button>
+                    <button
+                      className={`button button--${
+                        visibleCodeExample === 'UMD' ? 'info is-active' : 'info '
+                      }`}
+                      onClick={() => showCodeExample('UMD')}
+                    >
+                      UMD bundle support
```

Comment:
Can we make this one "Using on a website" or "With HTML"

I think there is probably a better term for this, but I can't think of it right now. I'm not sure how many "html devs" will actually know what UMD is though, so we should definitely try to use a phrase which is more approachable

Reply:
"HTML snippet example" or "CDN example"?

Reply:
I change it to "Install from CDN". How's it?

Reply:
I think it's still not entirely accurate in describing what it's doing. Technically installing from `npm` is installing from a CDN 😂 

But it's fine for now. Ship first and if we think of a more accurate term we can reword it